datafusion/sqllogictest/test_files/array.slt

```diff
@@ -265,10 +265,8 @@ AS VALUES
   (make_array([28, 29, 30], [31, 32, 33], [34, 35, 36], [28, 29, 30], [31, 32, 33], [34, 35, 36], [28, 29, 30], [31, 32, 33], [34, 35, 36], [28, 29, 30]), [28, 29, 30], [37, 38, 39], 10)
 ;
 
-query ?
+query error
 select [1, true, null]
```

Comment:
this is an error because `true` can't be coerced to an integer, right? FWIW I think that is fine and is consistent with the postgres rues:

```
postgres=# select array[1, true, null];
ERROR:  ARRAY types integer and boolean cannot be matched
LINE 1: select array[1, true, null];
```

Reply:
yes